vue-frontend/src/router/index.js

```diff
@@ -15,6 +17,11 @@ const routes = [
     name: "ContactScreen",
     component: ContactScreen,
   },
+  {
```

Comment:
Instead of changing component, we should add component in array list only if `IS_SHOW_JOBS` is true.